azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -109,13 +109,9 @@ Set-service -Name NetworkATC -StartupType Automatic
 
 ### Step 6: Add the Network ATC intent
 
-There are various intents that you can add. Identify the intent or intents you'd like using the examples in the next section.
+There are various intents that you can add. Identify the intent or intents you would like using the examples in the next section.
 
-To add the Network ATC intent, run the following command:
-
-```powershell
-Set-Service -Name NetworkATC -StartupType Automatic
-```
+To add the Network ATC intent, run the `Add-NetIntent` command with the appropriate options for the intent you want to deploy.
```

Comment:
@helmste - Given this is new content I think it's important that we detail the command. We can leave it as is.

Reply:
@ronmiab - The PowerShell command provided is completely wrong.  The command shown sets the service startup type to 'Automatic'.  This step is for customers to configure a new intent which is the 'Add-NetIntent' command.  The reason I did not spell out the command in its entirety is because there are MANY variations of the command depending on how the customer needs to configure their environment, and each has their own command which are detailed in the examples below.  If you want, I can provide more details such as:

Add-NetIntent <NameOfIntent> <Intent Types, Management, Compute, and/or Storage> <IntentOverrides> <VlanOverrides> 

or something similar.  This is a complex command, so I felt it was better served in the examples.